src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -149,8 +149,9 @@
   <data name="MinClientVersionAlreadyExist" xml:space="preserve">
     <value>MinClientVersion already exists in packages.config</value>
   </data>
-  <data name="MissingNuspec" xml:space="preserve">
-    <value>Nuspec file does not exist in package.</value>
+  <data name="Error_MissingNuspecFile" xml:space="preserve">
+    <value>The package doesn't contain nuspec file. {0}</value>
```

Comment:
Avoid contractions. 
Add an article. 

See my previous comment for a suggestion.